mivs/templates/emails/game_accepted_from_waitlist.txt

```diff
@@ -0,0 +1,22 @@
+{{ game.studio.primary_contact.first_name }},
```

Comment:
This is all just a copy/paste from the regular 'accepted' email. We probably want to change this somehow! @LowPolyCount if you want it changed in this PR, let me know what it should say and I'm happy to do it. :)

Reply:
I'm ok with changing the email later.